rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java

```diff
@@ -178,6 +178,48 @@ JobId submit(File job, Map<String, String> variables)
     JobId submit(URL job, Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
 
+    /**
+     * Submit a new job to the scheduler from the catalog with provided variables.
+     * <p>
+     * It will execute the tasks of the jobs as soon as resources are available.
+     * The job will be considered as finished once every tasks have finished
+     * (error or success). Thus, user could get the job result according to the
+     * precious result.
+     * <p>
+     * 
+     * @param bucketId
+     * @param workflowName
+     * @param variables
+     * @return
+     * @throws NotConnectedException
+     * @throws PermissionException
+     * @throws SubmissionClosedException
+     * @throws JobCreationException
+     */
+    JobId submitFromCatalog(String bucketId, String workflowName, Map<String, String> variables)
```

Comment:
I think it's important to include as well an option revision parameter.

Reply:
In the catalog client project the revision are not supported. I can implement it but in a separate moment as it will be more complex